netbeans-plugin/src/main/nbm/manifest.mf

```diff
@@ -0,0 +1,4 @@
+Manifest-Version: 1.0
```

Comment:
This file generated by netbeans automatically and it edit by netbeans. Project don't build without it.